spec/services/nytimes_service_spec.rb

```diff
@@ -9,16 +9,17 @@
 			end
 
 			expect(Article.first.title).to(
-				eq("California Imposes Water Restrictions, in First for State")
+				eq("California Drought Tests History of Endless Growth")
```

Comment:
Indent the first parameter one step more than the previous line.<br>Tab detected.